pyanaconda/modules/storage/installation.py

```diff
@@ -321,7 +321,7 @@ def _write_escrow_packets(self, storage, sysroot):
                 device.format.escrow(escrow_dir,
                                      backup_passphrase)
 
-        except (IOError, RuntimeError) as e:
+        except (OSError, RuntimeError) as e:
```

Comment:
I could not find where the error would come from, so I'm not sure if the replacement is correct. Perhaps @vojtechtrefny knows more?

Reply:
This probably comes from `util.makedirs`, which uses `os.makedirs`, which can raise `FileExistsError` (subclass of `OSError` so it's correct here).

Reply:
Thanks!